alpine-rails-buildkite/Dockerfile

```diff
@@ -0,0 +1,35 @@
+FROM dkubb/alpine-rails-nginx
+
+# Improve root state, intend to be pushed there
+WORKDIR /
+
+ENV RAILS_ENV=test                  \
+    PGBASE=/var/lib/postgresql      \
+    PGDATA=/var/lib/postgresql/data
+
+COPY etc /etc
+
+RUN rm -r /etc/sv/rails /etc/service/rails \
+ && /root/setup-directories.sh root r /etc/service/postgres /etc/service/buildkite \
+ && /root/setup-directories.sh postgres rw $PGBASE \
+ && ln -s -- /etc/sv/postgres /etc/service/postgres/run \
+ && ln -s -- /etc/sv/buildkite /etc/service/buildkite/run \
+ && echo '@testing-extra https://s3.amazonaws.com/alpine-packages/testing-extra' >> /etc/apk/repositories \
+ && apk add --update-cache imagemagick buildkite-agent@testing-extra phantomjs@testing-extra
+
+USER buildkite
+
+# Setup bundler for buildkite user
+RUN bundle config --global build.nokogiri '--use-system-libraries' \
+ && bundle config --global jobs '8'
+
+USER postgres
+
+# Setup user and database
+RUN pg_ctl initdb --pgdata $PGDATA \
```

Comment:
Given that the `PGDATA` environment is set globally, is it necessary to specify the `--pgdata` option?

Is it necessary to set `PGDATA` globally for every process running in the container, or should it be set in this RUN statement and in `/etc/sv/postgres` explicitly? I should note I have no strong preference either way, I just wanted to ask if it were a truly global thing or if it should be scoped.